lib/pf/Switch/Juniper/Junos_v15_x.pm

```diff
@@ -17,14 +17,14 @@ Developed and tested on Juniper ex2200 running on JUNOS 15.1
 use strict;
 use warnings;
 
-use base ('pf::Switch::Juniper::EX2200');
+use base ('pf::Switch::Juniper::Junos::Junos_v13_x');
 
 use pf::constants;
 use pf::config qw(
     $WIRED_802_1X
     $WIRED_MAC_AUTH
 );
-sub description { 'Juniper EX 2200 Series running Junos 15' }
+sub description { 'Junos v15.x' }
 
```

Comment:
Its defined, I believe it's defined in Junos_v12_x.pm modules and Junos_v12_x.pm is the parent of Junos_v13_x.pm which is the parent of Junos_v15_x.pm. So it can pull that logic from Junos_v12_x.pm.